packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -79,69 +79,69 @@ const ProductTile = (props) => {
     const styles = useMultiStyleConfig('ProductTile')
 
     return (
-        <Link
-            data-testid="product-tile"
-            {...styles.container}
-            to={productUrlBuilder({id: productId}, intl.local)}
-            {...rest}
-        >
-            <Box {...styles.imageWrapper}>
-                {image && (
-                    <AspectRatio {...styles.image}>
-                        <DynamicImage
-                            src={`${image.disBaseLink || image.link}[?sw={width}&q=60]`}
-                            widths={dynamicImageProps?.widths}
-                            imageProps={{
-                                alt: image.alt,
-                                ...dynamicImageProps?.imageProps
-                            }}
-                        />
-                    </AspectRatio>
-                )}
-
-                {enableFavourite && (
-                    <Box
-                        onClick={(e) => {
-                            // stop click event from bubbling
-                            // to avoid user from clicking the underlying
-                            // product while the favourite icon is disabled
-                            e.preventDefault()
+        <Box {...styles.container}>
+            <Link
+                data-testid="product-tile"
+                to={productUrlBuilder({id: productId}, intl.local)}
+                {...rest}
+            >
+                <Box {...styles.imageWrapper}>
+                    {image && (
+                        <AspectRatio {...styles.image}>
+                            <DynamicImage
+                                src={`${image.disBaseLink || image.link}[?sw={width}&q=60]`}
+                                widths={dynamicImageProps?.widths}
+                                imageProps={{
+                                    alt: image.alt,
+                                    ...dynamicImageProps?.imageProps
+                                }}
+                            />
+                        </AspectRatio>
+                    )}
+                </Box>
+
+                {/* Title */}
+                <Text {...styles.title}>{localizedProductName}</Text>
+
+                {/* Price */}
+                <Text {...styles.price} data-testid="product-tile-price">
+                    {hitType === 'set' &&
+                        intl.formatMessage({
+                            id: 'product_tile.label.starting_at_price',
+                            defaultMessage: 'Starting at'
+                        })}{' '}
+                    {intl.formatNumber(price, {
+                        style: 'currency',
+                        currency: currency || activeCurrency
+                    })}
```

Comment:
```suggestion
                    
                {hitType === 'set'
                    ? intl.formatMessage(
                          {
                              id: 'product_tile.label.starting_at_price',
                              defaultMessage: 'Starting at {price}'
                          },
                          {
                              price: intl.formatNumber(price, {
                                  style: 'currency',
                                  currency: currency || activeCurrency
                              })
                          }
                      )
                    : intl.formatNumber(price, {
                          style: 'currency',
                          currency: currency || activeCurrency
                      })}
```
Price is part of the message! (Gotta support languages that do `{price} starting at`)